docs/config.json

```
@@ -27,6 +27,13 @@
       "path": "docs/RESOURCES.md",
       "title": "Useful Common Lisp resources",
       "blurb": "A collection of useful resources to help you master Common Lisp"
-    }
+    },
+        {
```

Just in case you care:

```suggestion
    {
```

---

Thanks.